### PR TITLE
Added Instructions to Readme File

### PR DESCRIPTION
**Pull request by alexjhawk**
Updated the readme.md file to contain information about the demo, and instructions to download, install and use the demo.

### CODE REVIEW

**Comment by alexjhawk** [comment]
I think this is now ready for review. 

For now, there has been no official release of the demo, so the "resource pack" for the demo (the files needed for this setup), are found in a folder under the solution center shared files.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,115 @@
...
+
+- *Flexy 205*: [https://websupport.ewon.biz/sites/default/files/ig-0028-00-en-installation-guide-for-flexy-205.pdf#page=22](https://websupport.ewon.biz/sites/default/files/ig-0028-00-en-installation-guide-for-flexy-205.pdf#page=22 "https://websupport.ewon.biz/sites/default/files/ig-0028-00-en-installation-guide-for-flexy-205.pdf#page=22")
+- *Flexy Base Unit*: [https://websupport.ewon.biz/sites/default/files/ig-014-0-en-ewon_flexy_-_base_units.pdf#page=28](https://websupport.ewon.biz/sites/default/files/ig-014-0-en-ewon_flexy_-_base_units.pdf#page=28 "https://websupport.ewon.biz/sites/default/files/ig-014-0-en-ewon_flexy_-_base_units.pdf#page=28")
+- *Cosy 131*: [https://websupport.ewon.biz/sites/default/files/ig-0022-00-en-installation-guide-for-cosy-131.pdf#page=24](https://websupport.ewon.biz/sites/default/files/ig-0022-00-en-installation-guide-for-cosy-131.pdf#page=24 "https://websupport.ewon.biz/sites/default/files/ig-0022-00-en-installation-guide-for-cosy-131.pdf#page=24")
```
Comment (TomKimsey) [inline README.md]: Cozy is not compatible with datamailbox.  Only the Flexy will work with ignition.

Comment (alexjhawk) [reply README.md]: Removed Cozy Link

File: README.md [hunk 2]
```diff
@@ -1,3 +1,114 @@
-# FlexyIgnitionDemo
-A Java Application for Flexy, providing a Multi-Dashboard Demo on the Ignition platform from Inductive Automation
+# HMS Ewon Ignition Integration Demo #
 
+----------
+
+## About ##
+HMS Ewon Ignition Integration Demo is a small suite of applications that showcase the features of integration between HMS Industrial Network's Ewon platform, and the Inductive Automation Ignition platform.
+
+HMS Ewon Ignition Integration Demo was developed by the HMS Solution Center using the Ewon Java Library. For more information on this demo and developing applications with the Ewon Java Library, please contact us at [us-services@hms.se](mailto:us-services@hms.se "us-services@hms.se") and/or visit the Ewon Developer website at [https://developer.ewon.biz/](https://developer.ewon.biz/ "https://developer.ewon.biz/")
```
Comment (TomKimsey) [inline README.md]: There are multiple solution centers so you could just say developed by HMS or state that its the North America's Solution Center or whatever the current official name is.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,114 @@
...
+
+**Step 3:** After Ignition is installed, the Ignition Gateway web page will open and guide you through commissioning Ignition. If the Ignition Gateway web page does not automatically open, using your browser, navigate to [http://localhost:8088](http://localhost:8088 "http://localhost:8088"). Take note of the username and password you set during commissioning, as this will be required in later setup steps.
+
+**Step 4:** After Ignition is commissioned, you will be redirected to the Ignition Gateway homepage. Take note of the URL in your browser's address bar as this will be required in later setup steps.
+
+**Step 5:** Download the HMS Ewon Connector from the EwonConnector releases page, found here: [https://github.com/hms-networks/EwonConnector/releases](https://github.com/hms-networks/EwonConnector/releases "https://github.com/hms-networks/EwonConnector/releases"). To download, expand the `Assets` menu under the latest Ignition 8 release, and click on the first file, commonly named `EwonConnection-X.Y.Z.zip`, where X.Y.Z is the version number.
```
Comment (TomKimsey) [inline README.md]: Maybe say they should be grabbing the latest release and the minimum supported version is 1.1.7

File: README.md [hunk 4]
```diff
@@ -1,3 +1,114 @@
...
+
+## Issues/Problems ##
+If you experience a problem or bug, please check the Ewon Ignition Integration Demo Wiki for information, found here: [https://github.com/hms-networks/FlexyIgnitionDemo/wiki](https://github.com/hms-networks/FlexyIgnitionDemo/wiki "hms-networks/FlexyIgnitionDemo/wiki")  
+
+If no information is available on the Ewon Ignition Integration Demo Wiki, or if your issue persists, please create a GitHub issue, found here: [https://github.com/hms-networks/FlexyIgnitionDemo/issues](https://github.com/hms-networks/FlexyIgnitionDemo/issues "hms-networks/FlexyIgnitionDemo/issues")  
+
+## Getting Started ##
+To complete installation, please perform all the setup steps in the sections below! To begin, the following items are required:  
+
+- HMS Ewon Product
```
Comment (TomKimsey) [inline README.md]: trailing ws

File: README.md [hunk 5]
```diff
@@ -1,3 +1,114 @@
...
+
+HMS Ewon Ignition Integration Demo was developed by the HMS Solution Center using the Ewon Java Library. For more information on this demo and developing applications with the Ewon Java Library, please contact us at [us-services@hms.se](mailto:us-services@hms.se "us-services@hms.se") and/or visit the Ewon Developer website at [https://developer.ewon.biz/](https://developer.ewon.biz/ "https://developer.ewon.biz/")
+
+## Issues/Problems ##
+If you experience a problem or bug, please check the Ewon Ignition Integration Demo Wiki for information, found here: [https://github.com/hms-networks/FlexyIgnitionDemo/wiki](https://github.com/hms-networks/FlexyIgnitionDemo/wiki "hms-networks/FlexyIgnitionDemo/wiki")  
+
+If no information is available on the Ewon Ignition Integration Demo Wiki, or if your issue persists, please create a GitHub issue, found here: [https://github.com/hms-networks/FlexyIgnitionDemo/issues](https://github.com/hms-networks/FlexyIgnitionDemo/issues "hms-networks/FlexyIgnitionDemo/issues")  
+
+## Getting Started ##
+To complete installation, please perform all the setup steps in the sections below! To begin, the following items are required:
```
Comment (TomKimsey) [inline README.md]: trailing ws

File: README.md [hunk 6]
```diff
@@ -1,3 +1,114 @@
...
 
+----------
+
+## About ##
+HMS Ewon Ignition Integration Demo is a small suite of applications that showcase the features of integration between HMS Industrial Network's Ewon platform, and the Inductive Automation Ignition platform.
+
+HMS Ewon Ignition Integration Demo was developed by the HMS Solution Center using the Ewon Java Library. For more information on this demo and developing applications with the Ewon Java Library, please contact us at [us-services@hms.se](mailto:us-services@hms.se "us-services@hms.se") and/or visit the Ewon Developer website at [https://developer.ewon.biz/](https://developer.ewon.biz/ "https://developer.ewon.biz/")
+
+## Issues/Problems ##
+If you experience a problem or bug, please check the Ewon Ignition Integration Demo Wiki for information, found here: [https://github.com/hms-networks/FlexyIgnitionDemo/wiki](https://github.com/hms-networks/FlexyIgnitionDemo/wiki "hms-networks/FlexyIgnitionDemo/wiki")
```
Comment (TomKimsey) [inline README.md]: trailing ws

File: README.md [hunk 7]
```diff
@@ -1,3 +1,114 @@
...
+
+HMS Ewon Ignition Integration Demo was developed by the HMS Solution Center using the Ewon Java Library. For more information on this demo and developing applications with the Ewon Java Library, please contact us at [us-services@hms.se](mailto:us-services@hms.se "us-services@hms.se") and/or visit the Ewon Developer website at [https://developer.ewon.biz/](https://developer.ewon.biz/ "https://developer.ewon.biz/")
+
+## Issues/Problems ##
+If you experience a problem or bug, please check the Ewon Ignition Integration Demo Wiki for information, found here: [https://github.com/hms-networks/FlexyIgnitionDemo/wiki](https://github.com/hms-networks/FlexyIgnitionDemo/wiki "hms-networks/FlexyIgnitionDemo/wiki")  
+
+If no information is available on the Ewon Ignition Integration Demo Wiki, or if your issue persists, please create a GitHub issue, found here: [https://github.com/hms-networks/FlexyIgnitionDemo/issues](https://github.com/hms-networks/FlexyIgnitionDemo/issues "hms-networks/FlexyIgnitionDemo/issues")
```
Comment (TomKimsey) [inline README.md]: trailing ws

File: README.md [hunk 8]
```diff
@@ -1,3 +1,114 @@
...
+
+**Step 5:** Download the HMS Ewon Connector from the EwonConnector releases page, found here: [https://github.com/hms-networks/EwonConnector/releases](https://github.com/hms-networks/EwonConnector/releases "https://github.com/hms-networks/EwonConnector/releases"). To download, expand the `Assets` menu under the latest Ignition 8 release, and click on the first file, commonly named `EwonConnection-X.Y.Z.zip`, where X.Y.Z is the version number.
+
+**Step 6:** After the download is complete, install the HMS Ewon Connector by going to the `Config` tab in Ignition, clicking `Modules` and then choosing `Install or Upgrade a Module...`. Click `Choose File` and select the file you've just downloaded, then click `Install`.
+
+**Step 7:** After installation of the HMS Ewon Connector, browse to the Ewon Connector configuration by going to the `Config` tab in Ignition, and clicking `Ewon Connector`, located under the `Tags` heading. *NOTE: the default name "Ewon" should NOT be changed*.
+
+**Step 8:** Ensure that the `Enabled` check-box is checked.
```
Comment (TomKimsey) [inline README.md]: trailing ws

File: README.md [hunk 9]
```diff
@@ -1,3 +1,114 @@
...
+
+**Step 10:** [Optional] Click the down arrow, located to the right of `Launch Fullscreen`, and click `Create Shortcut` to add a demo shortcut to your desktop for convenience.
+
+## Starting the Demo ##
+
+**Step 1:** Launch Vision Client Launcher by clicking it's icon in your operating system's main menu.
+
+* *Windows:* Start Button
+* *macOS:* Launchpad
+	* *Note: macOS Catalina (10.15) does not automatically add new applications to Launchpad. Open `Finder`, and click `Applications` to locate applications. To add an application to Launchpad, drag it to the Launchpad icon.*
```
Comment (TomKimsey) [inline README.md]: Do we really need to be this explicit in a readme?  If they cant use their OS thats really not our problem, especially when they are using an OS that is still in beta.

Comment (TomKimsey) [reply README.md]: I think its fine to just say launch Vision Client Launcher and leave it at that, but its fine to keep os specific instructions in.

File: README.md [hunk 10]
```diff
@@ -1,3 +1,114 @@
...
+
+**Step 3:** Assign the project a name of your choice, and click `Import`.
+
+**Step 4:** After successfully importing the Vision project, click the `Home` tab.
+
+**Step 5:** Download Vision Client Launcher by clicking `Download Vision Client Launcher`, and choosing your operating system.
+
+**Step 6:** After the download is complete, open the downloaded file, and follow the installation procedure to completion. The default settings are satisfactory, but you may modify them as desired.
+
+**Step 7:** After the installation is complete, open Vision Client Launcher if you have not done so already, and click `Add Vision Client`. For more information on location/opening Vision Client Launcher, see Starting the Demo step 1.
```
Comment (TomKimsey) [inline README.md]: Should step be Step?

File: README.md [hunk 11]
```diff
@@ -1,3 +1,114 @@
...
+
+**Step 7:** After installation of the HMS Ewon Connector, browse to the Ewon Connector configuration by going to the `Config` tab in Ignition, and clicking `Ewon Connector`, located under the `Tags` heading. *NOTE: the default name "Ewon" should NOT be changed*.
+
+**Step 8:** Ensure that the `Enabled` check-box is checked.  
+
+**Step 9:** Set the `Live Data Poll Rate in Seconds` value to 1 second.
+
+**Step 10:** Ensure that the `Read all values in realtime` check-box is checked.
+
+**Step 11:** Under `Ewon Account Information`, input your Talk2M account information. All fields are required. Please note that you may need to select the `Change Password?` option to enable the password fields. If you do not have a Talk2M/Ewon API Key, you may request one here: [https://developer.ewon.biz/registration](https://developer.ewon.biz/registration "https://developer.ewon.biz/registration").
```
Comment (TomKimsey) [inline README.md]: This almost should be in the Ewon Setup area as it takes a bit of time to get the DevID.  If they complete this step early they may have their DevID by the time they get here.

File: README.md [hunk 12]
```diff
@@ -1,3 +1,114 @@
...
+- *Others*: [https://websupport.ewon.biz/support](https://websupport.ewon.biz/support "https://websupport.ewon.biz/support")
+
+**Step 2:** Using a web browser, navigate to your Ewon's web page and select the `Users` tab, located under `Setup`.
+
+**Step 3:** Click `Add`, then create a username and password of your choice. Take note of this username and password, as it will be required in later setup steps. If you'd like, you may populate the `First Name`, `Last Name` and/or `Information` fields. IMPORTANT: Do not modify the rights of the new user.
+
+**Step 4:** Click `Add User` to complete the addition of the new user account.
+
+## Ignition Setup ##
+*Information for setup of Ignition (steps 1 - 4) can also be found here: [https://docs.inductiveautomation.com/display/DOC80/Installing+and+Upgrading](https://docs.inductiveautomation.com/display/DOC80/Installing+and+Upgrading "https://docs.inductiveautomation.com/display/DOC80/Installing+and+Upgrading")*
```
Comment (TomKimsey) [inline README.md]: Should Steps be capitalized as they are specific steps?

File: README.md (hunk 11 above)
Comment (alexjhawk) [reply README.md]: Moved to the list of "required items" under Getting Started

File: README.md [hunk 13]
```diff
@@ -1,3 +1,116 @@
...
+
+**Step 1:** Connect your computer to the LAN of your Ewon device, and open an FTP connection to your Ewon device. To create an FTP connection, using your file manager (i.e. Windows Explorer), enter `ftp://[EWON IP HERE]/` in the address bar, and hit enter. If you don't know the IP address of your Ewon device, you can locate it using eBuddy.
+
+**Step 2:** Copy the files `EwonIgnitionDemoApp.jar` and `jvmrun` from the demo resource pack into the usr folder on your Ewon device using the FTP connection you've just opened.
+
+**Step 3:** Reboot your Ewon device by navigating to its web page, clicking the `Setup` tab, pressing `Reboot` and then pressing `REBOOT` once more.
+
+## Demo Dashboard Setup ##
+
+**Step 1:** Browse to your Ignition Gateway's web page using the URL from step 4 of Ignition Setup. Click the `Config` tab, and open `Projects`.
```
Comment (TomKimsey) [inline README.md]: **S**tep

Sorry I missed this one.

**Pull request merged**